src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Public.Data/DataSetVersionsController.cs

```
@@ -52,11 +53,16 @@ public async Task<ActionResult<DataSetVersionSummaryViewModel>> CreateNextVersio
         [FromBody] NextDataSetVersionCreateRequest nextDataSetVersionCreateRequest,
         CancellationToken cancellationToken)
     {
+        var dataSetVersionToPatch = nextDataSetVersionCreateRequest.DataSetVersionToPatch is not null 
+            ? SemVersion.Parse(nextDataSetVersionCreateRequest.DataSetVersionToPatch, SemVersionStyles.OptionalMinorPatch
```

Won't this throw an exception if the version can not be parsed instead of returning a failure status code and message?
Also, when checking a string, it is usual to check `IsNullOrEmpty` especially on a DTO which may indeed be automatically populated with string.empty by the deserialiser.
Lastly, please put the `: null` on the next line to indicate that it goes with the `?` on Line 57.